README.md

```diff
@@ -321,6 +321,8 @@ const matrixClient = sdk.createClient({
 await matrixClient.initRustCrypto();
 ```
 
+Note that by default it will attempt to use the Indexed DB provided by the browser as a crypto store. If running outside the browser, you will need to pass [an options object](https://matrix-org.github.io/matrix-js-sdk/classes/matrix.MatrixClient.html#initrustcrypto) which includes `useIndexedDB: false`, to use an ephemeral in-memory store instead.
```

Comment:
Thanks for this.  I think if we're going to suggest this, we'd also better point out the pitfalls of not using a persistent store: specifically, that you'll need to create a new device on the server side (with `MatrixClient.loginRequest`) each time your application starts.

Reply:
OK thanks for the info - I've amended the PR to include that.